ui/src/components/RollupRuleEditor.js

```diff
@@ -123,21 +134,27 @@ const TargetsEditorBase = props => {
                 <Icon type="delete" />
               </a>
             }>
-            <label>Metric Name</label>
+            <label>New Metric Name</label>
```

Comment:
Do you want to call this `New Rollup Metric Name`?

Reply:
Sure, that works. Might be worth to user test these types of things. It's hard to say with our tunnel vision now lol

Reply:
Agreed =p